_data/committers.yml

```
@@ -424,6 +424,10 @@
   role: Committer
   alias: wayne
   affiliation: Greptime
+- name: Sarah Gilmore
+  role: Committer
+  alias: sgilmore10
```

This is your Apache ID not GitHub ID: https://people.apache.org/phonebook.html?uid=sgilmore

```suggestion
  alias: sgilmore
```

---

My apologies! I misunderstood.